src/OptionSettlementEngine.sol

```diff
@@ -84,6 +72,18 @@ contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
         mapping(uint96 => BucketExerciseState) bucketExerciseStates;
     }
 
+    /**
+     * @notice Claims can be used to write multiple times. This struct is used to keep track
+     * of how many options are written against a claim in each bucket, in order to
```

Comment:
Rather than "against a claim in each bucket", "from a claim into each bucket"

Reply:
Done